examples/java/src/main/java/glide/examples/ClusterExample.java

```diff
@@ -0,0 +1,160 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.examples;
+
+import static glide.api.models.configuration.RequestRoutingConfiguration.SimpleMultiNodeRoute.*;
+
+import glide.api.GlideClusterClient;
+import glide.api.logging.Logger;
+import glide.api.models.ClusterValue;
+import glide.api.models.commands.InfoOptions;
+import glide.api.models.configuration.GlideClusterClientConfiguration;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.ConnectionException;
+import glide.api.models.exceptions.TimeoutException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class ClusterExample {
+
+    /**
+     * Creates and returns a GlideClusterClient instance. This function initializes a
+     * GlideClusterClient with the provided list of nodes.
+     *
+     * @return <code>GlideClusterClient</code> An instance of <code>GlideClusterClient</code>
+     *     connected to the discovered nodes.
+     */
+    public static GlideClusterClient createClient() throws ExecutionException, InterruptedException {
+        String host = "localhost";
+        Integer port1 = 7001;
+        Integer port2 = 7002;
+        Integer port3 = 7003;
+        Integer port4 = 7004;
+        Integer port5 = 7005;
+        Integer port6 = 7006;
```

Comment:
In Python's example, it only uses `localhost` and `6379`. However, in Java's `README.md`, it provides the Cluster Valkey example with 6 ports. Should I follow Python's example or should I follow README.md?

Reply:
Should work both. I think the best option is to show example with one port, but tell that GLIDE is able to detect all cluster nodes and connect to them automatically.

Reply:
Ok. I will update the example to use only one port.
I will add a comment to state that "Multiple ports can be configured and GLIDE is able to detect all cluster nodes and connect to them automatically".